addons/web_dashboard/models/res_users.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+from datetime import datetime, timedelta
+
+from openerp import models, api, _
+from openerp.exceptions import RedirectWarning
+from openerp.tools.misc import DEFAULT_SERVER_DATETIME_FORMAT
+
+
+class Users(models.Model):
+    _inherit = 'res.users'
+
+    def web_dashboard_get_users_by_state(self):
```

Comment:
I think @antonylesuisse asked you to make a method on res.users... but for me this one belongs more to a controller (only marshmalling).